pkg/sink/sink.go

```diff
@@ -376,6 +400,8 @@ func (r Sink) processTrigger(t triggersv1.Trigger, request *http.Request, event
 		return
 	}
 	go r.recordResourceCreation(resources)
+	r.sendCloudEvents(request.Header, *el, eventID, events.TriggerProcessingSuccessfulV1)
```

Comment:
Are we planning to send events of processTrigger as well ?

/cc @dibyom

Reply:
TriggerProcessing errors aren't error most of the time. It's because of filters, etc. So we are sending only Successful Event if a trigger succeed.

Reply:
Ah okok could you add this as well 

```
r.emitEvents(r.EventRecorder, el, events.TriggerProcessingSuccessfulV1, nil)
```
which handles for K8s

Reply:
Sure.